src/pages/UsersPage.tsx

```diff
@@ -193,6 +178,45 @@ export function UsersPage() {
               deleteUserFunction({ id: deletedUserId! });
             }}
           />
+
+          <Modal
+            title="User invited successfully"
+            onClose={() => {
+              setInviteLink(undefined);
+            }}
+            open={!!inviteLink}
+            bodyClass=""
+            size={"small"}
+          >
+            <div className="flex flex-col bg-white p-4">
+              <div className="p-2 text-sm">
+                User has been invited successfully. An email has been sent to
```

Comment:
Why do we have this?  The invite link is considered a password and shouldn't be shared with anyone.

Reply:
Thinking about this more, I think we should keep it in